### PR TITLE
Add grunt watch and local server for easier upgrading of documentation

### PR DESCRIPTION
**Pull request by wildermuthn**
Since the documentation changed, it has been harder to find what I'm looking for in Ramda. I was thinking about updating the documentation, but noticed that the Gruntfile is missing watch and expess/connect. It's saves a lot of time to have the browser reloading automatically as changes are made to the html templates and js files. 

'grunt serve' now builds the documentation (github pages), then watches for changes. If it sees changes, the server reloads the page automatically. Standard web-dev stuff. Hope it helps!

### CODE REVIEW

**Comment by wildermuthn** [comment]
Looks like indentation comes through strange via github.

File: Gruntfile.js [hunk 1]
```diff
@@ -126,7 +126,32 @@ module.exports = function(grunt) {
...
+									bases: ['./dist/gh-pages'], 
+									livereload: true
+							}
+					}
+				},
+				open: {
+					docs: {
+						path: 'http://localhost:<%= express.docs.options.port%>/docs'
+					}
+				}
```
Comment (davidchambers) [inline Gruntfile.js]: Woah! Spaces rather than tabs, please. :)

File: Gruntfile.js [hunk 2]
```diff
@@ -195,4 +223,9 @@ module.exports = function(grunt) {
         'gh-pages',
         'push-gh-pages'
     ]);
+    grunt.registerTask('serve', [
+        'express',
+				'open',
```
Comment (davidchambers) [inline Gruntfile.js]: :triangular_ruler:

**Comment by wildermuthn** [comment]
Too many weird problems. I'll just issue another PR later.

**Pull request closed without merging**